tests/ZendService/Amazon/Ec2/AbstractTest.php

```diff
@@ -32,7 +33,7 @@ public function testSetRegion()
     public function testSetInvalidRegionThrowsException()
     {
         $ec2 = new TestAmazonAbstract('TestAccessKey', 'TestSecretKey');
-        $this->setExpectedException(
+        $this->expectException(
```

Comment:
`setExpectedException` was replaced by `expectException()` + `expectExceptionMessage()` in phpunit